Testscripts/Linux/XDPForwardingTest.sh

```diff
@@ -152,16 +179,29 @@ for mode in ${modes[@]}; do
 		#iptables -t nat -A PREROUTING -i eth1 -p udp --dport 9 -j DNAT --to 10.0.1.6:9999
 		receiverIP=$((ssh $receiver "/sbin/ifconfig ${nicName} | grep 'inet' | cut -d: -f2") | awk '{print $2}')
 
-		ssh ${forwarder} "iptables -t nat -A PREROUTING -i ${nicName} -p udp --dport 9 -j DNAT --to ${receiverIP}:9999"
+		ssh ${forwarder} "iptables -t nat -A PREROUTING -i ${nicName} -p udp -j DNAT --to-destination ${receiverIP}:9999"
 		ssh ${forwarder} "iptables -t nat -A POSTROUTING -j MASQUERADE"
 		LogMsg "${forwarder} vm setup completed for ip tables"
 	fi
 
 	# Start pktgen on Sender
 	forwarderSecondMAC=$((ssh ${forwarder} "ip link show ${nicName}") | grep ether | awk '{print $2}')
-	LogMsg "Starting pktgen on ${sender}"
-	start_pktgen ${sender} ${cores} ${pktgenDir} ${nicName} ${forwarderSecondMAC} ${forwarderSecondIP} ${packetCount}
-	sleep 5
+	LogMsg "Starting dpdk-testpmd on ${sender}"
+	#start_pktgen ${sender} ${cores} ${pktgenDir} ${nicName} ${forwarderSecondMAC} ${forwarderSecondIP} ${packetCount}
```

Comment:
Don't leave the code you are not going to use.

Reply:
done!